src/KafkaFlow.Retry.SqlServer/Deploy/01 - Create_Tables.sql

```diff
@@ -130,6 +130,41 @@ BEGIN
 END
 GO
 
+IF(NOT EXISTS(SELECT *
```

Comment:
In future development, we can change those `SELECT *` to `SELECT 1`. A very small improvement, but actually improves the whole script execution because the column values of `sys.types` is not actually returned by the database engine, and the select query returns faster.